frontend/dockerfile/instructions/parse.go

```diff
@@ -84,6 +92,11 @@ func ParseInstruction(node *parser.Node) (v interface{}, err error) {
 	case command.Copy:
 		return parseCopy(req)
 	case command.From:
+		if !linter.ValidateStageNameCasing(req.args) {
+			location := linter.FirstNodeRange(node)
+			stageNameCasing := linter.LinterRules[linter.RuleStageNameCasing]
+			warn(stageNameCasing.Short, "", [][]byte{[]byte(stageNameCasing.Description)}, location)
```

Comment:
I'd like this to be so the function that triggers the warning only accepts the type from the `linter` pkg and the values defined there are included automatically instead of reading from a map and then passing individual values. In here, only location and maybe something about the textual value of "node" is added.